samples/matter/common/src/matter_init.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include <app/clusters/network-commissioning/network-commissioning.h>
+#include <lib/core/Optional.h>
+#include <lib/support/Variant.h>
+#include <platform/PlatformManager.h>
+#if CONFIG_CHIP_FACTORY_DATA
+#include <platform/nrfconnect/FactoryDataProvider.h>
+#else
+#include <platform/nrfconnect/DeviceInstanceInfoProviderImpl.h>
+#endif
+
+namespace Nordic
```

Comment:
Should we have a task to move all common files to this namespace? I mean this should probably be unified, right?